HW02_A_Graph-Fails.Rmd

```diff
@@ -61,6 +80,31 @@ ggplot() +
 
 ```
 
+**The attribute *alpha* is a value between 0 and 1, and it sets opacity of the point. Setting it to 2 is meaningless. You can set the attribute *size* to make the point bigger.** 
+
+**We set position of legend by setting a point pair where values of x and y are between 0 and 1, not following the text on the axis. Setting *legend.title = element_blank()* removes legend title, and setting *legend.position = "none"* removes legend.**
+
+### Corrected Graph 3 - Legend on the Top Without Title
+
+```{r}
+mpg %>% 
+ggplot() + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = class), size = 2) + 
+  theme(legend.direction = "horizontal") + 
+  theme(legend.position = c(0.67, 0.86), legend.title = element_blank())

```

Comment:
Maybe you could add these two lines together into a single theme() function? That way everything's in one place.

Reply:
Yes. This is a good suggestion.